src/cdk/collections/selection.ts

```diff
@@ -116,6 +116,13 @@ export class SelectionModel<T> {
     }
   }
 
+  /**
+   * Determines whether multiple values can be selected.
```

Comment:
One last nit: the description should be "Gets whether multiple values can be selected."